packages/prisma-generator-proto/src/generate/hooks.ts

```diff
@@ -1,17 +1,62 @@
 import { DMMF } from "@prisma/generator-helper";
 
 function generateSingleHook(model: DMMF.Model): string {
+  const initialValue: { [key: string]: string | number } = {};
+  for (const field of model.fields) {
+    if (field.kind === "scalar" && field.name !== "id") {
+      initialValue[field.name] = "";
```

Comment:
Let's assign corresponding initial value for different field types (`field.type`). You can find the list of Prisma scalar types here: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#model-field-scalar-types
Let's ignore `Decimal`, `DateTime`, `Json` and `Bytes` types for now.